apps/odk_publish/forms.py

```diff
@@ -293,11 +293,12 @@ class ProjectForm(PlatformFormMixin, forms.ModelForm):
 
     class Meta:
         model = Project
-        fields = ["name", "central_server", "template_variables"]
+        fields = ["name", "central_server", "template_variables", "app_language"]
         widgets = {
             "name": TextInput,
             "central_server": Select,
             "template_variables": CheckboxSelectMultiple,
+            "app_language": Select(attrs={"class": "!w-30"}),
```

Comment:
Nice touch, the dropdown looks good to me:

<img width="464" alt="Screenshot 2025-04-21 at 2 15 02 PM" src="https://github.com/user-attachments/assets/a90b04d2-8b54-4d47-a9a5-aa236a53f861" />